Riot/Modules/Room/DataSources/RoomDataSource.m

```diff
@@ -1095,83 +1087,6 @@ - (void)updateStatusInfo
     }
 }
 
-#pragma mark - Room creation intro cell
-
-- (BOOL)canShowRoomCreationIntroCell
-{
-    NSString* userId = self.mxSession.myUser.userId;
-
-    if (!userId || !self.isLive || self.isPeeking)
```

Comment:
Apart from checking against `userId`, which we no longer require (i.e. all users should see the header), it wasn't obvious whether there is any value in keeping the `isLive` and `isPeeking` checks. I assume that the "m.room.create" event is desirable in all cases.